src/WebClient.spec.js

```diff
@@ -494,6 +485,23 @@ describe('WebClient', function () {
     });
   });
 
+  describe('apiCall() - when using static headers', function () {
+    it('should include static headers on api request', function () {
+      const client = new WebClient(token, { headers: { 'X-XYZ': 'value' } });
+      const scope = nock('https://slack.com')
+        .post(/api/)
+        .reply(200, { ok: true })
+        .on('request', function (req, interceptor, body) {
```

Comment:
this is a pretty clever way to check for headers, cool!

i think it would be a little more straight-froward if you were to use nock's `reqheaders` option, which was designed for this purpose. check it out here: https://github.com/nock/nock#specifying-request-headers

Reply:
Ah, thank you! Didn't notice that when looking over Nock :+1: